src/pages/OrderPage.tsx

```diff
@@ -0,0 +1,28 @@
+import HeaderBar from "../components/HeaderBar/HeaderBar"
+import { useParams } from "react-router-dom"
+import { Container, Row, Col } from "react-bootstrap";
+
+interface Params {
+    collectionId: string
+}
+
+const OrderPage = () => {
+    const params = useParams<Params>();
+    console.log(params.collectionId)
+    //  Will use collectionId to call orders API
+    
+    return (
+        <>
```

Comment:
I like these but some people might prefer React.Fragment I'm not sure

Reply:
There is no real reason to use a React.Fragment anymore - generally, if you see them then it's because the code was written before the short-form fragment was added